### PR TITLE
schedule DAGs

### PR DESCRIPTION
**Pull request by jacobthill**
Schedules DAGs.

### CODE REVIEW

File: catalogs/bodleian.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 metadata:
   version: 1
   data_path: bodleian
-  schedule: "40 8 * * *"
+  schedule: "30 11 01 Jan,Apr,Jul,Oct"
```
Comment (jacobthill) [inline catalogs/bodleian.yaml:4]: This is supposed to be 11:30am (assuming this is Pacific time) the 1st of Jan, Apr, Jul, Oct

**Comment by aaron-collier** [comment]
@jacobthill airflow appears not to support this format:

```
Broken DAG: [/opt/airflow/dlme_airflow/dags/harvest_catalog.py] Invalid timetable expression: Exactly 5 or 6 columns has to be specified for iterator expression.
```

I'll make a commit with the proper numbering to make this happen correctly.

**Review by aaron-collier — CHANGES_REQUESTED** [review]
RC to hold this, see comment.

File: catalogs/bodleian.yaml (hunk 1 above)
Comment (aaron-collier) [reply catalogs/bodleian.yaml:4]: @jacobthill you left off the "day of the week" notation, so this should be:

```
30 11 01 Jan,Apr,Jul,Oct *
```

As well as the other ones. That will fix the error on startup.

**Review by aaron-collier — CHANGES_REQUESTED** [review]

**Review by aaron-collier — APPROVED** [review]

**Pull request merged**